# Hekateros hekateros_control/IsCalibrated service

---
bool is_calibrated    # robot is [not] calibrated
